packages/formik/src/types.tsx

```diff
@@ -267,11 +269,11 @@ export type GenericFieldHTMLAttributes =
   | JSX.IntrinsicElements['textarea'];
 
 /** Field metadata */
-export interface FieldMetaProps<Value> {
+export interface FieldMetaProps<Value, Error> {
   /** Value of the field */
   value: Value;
   /** Error message of the field */
-  error?: string;
+  error?: Error;
```

Comment:
Is this the correct type?  I just ran across an error in the types for `FieldMetaProps` when using `useField`.  Currently, the type is `string` as it is here, but it actually seems to be able to be either `string` or the shape of `Value`.  So, I was contemplating submitting a PR that changed the type to: `error?: FormikErrors<Value> | string`.  But then I ran across this PR and thought it could be done here.

The `Error` generic type parmater you have is nice for customizing the `error` that is returned (either string or some custom object that the user passes in), but I think the default of the generic `Error` type parameter should not be just `string` but `FormikErrors<Value> | string`?  

In reality, if the `Value` you use for a Formik field is not a plain string, the error object you get back matches the shape of the `Value` (I think), so it might be more helpful.  It still seems useful to have your `Error` generic type parameter, but I think the default type should be different (maybe changed to `FormikErrors<Value> | string`)...

Reply:
@BrentFarese thanks for suggestions! do you mind to create reproducible example on error you'd find on `useFiled` feel free to use codesandbox example from above, or any you'd like. I'm gonna try your suggested default value as soon as will have some free time, but things are not super fast in this repo, so I guess no rush